checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallInference.java

```diff
@@ -670,11 +767,46 @@ private void computeOwningFromInvocation(Set<Obligation> obligations, Node invoc
       // @EnsuresCalledMethods annotations on constructors as we have not observed them in practice.
       inferOwningParamsViaOwnershipTransfer(obligations, invocation);
     } else if (invocation instanceof MethodInvocationNode) {
+      computeMustCallAliasFromThisOrSuperCall(obligations, (MethodInvocationNode) invocation);
       inferOwningParamsViaOwnershipTransfer(obligations, invocation);
       computeOwningForArgsOrReceiverOfCall(obligations, (MethodInvocationNode) invocation);
     }
   }
 
+  /**
+   * Adds @MustCallAlias annotation to a parameter of the current method when it is passed in
+   * a @MustCallAlias position in a {@literal this} or {@literal super} call.
```

Comment:
Add the word "constructor" in this comment somewhere, because right now I think it's a bit confusingly worded. In particular, it's possible to have `super` calls that are not to the superclass constructor: for example, to access an overridden method.